domain/application/service/application.go

```diff
@@ -1468,3 +1474,11 @@ func (s *Service) ResolveControllerCharmDownload(ctx context.Context, resolve ap
 		ArchivePath: result.UniqueName,
 	}, nil
 }
+
+// GetApplicationsForRevisionUpdater returns all the applications for the
+// revision updater. This will only return charmhub charms, for applications
+// that are alive.
+// This will return an empty slice if there are no applications.
+func (s *Service) GetApplicationsForRevisionUpdater(ctx context.Context) ([]application.RevisionUpdaterApplication, error) {
+	return s.st.GetApplicationsForRevisionUpdater(ctx)
+}
```

Comment:
I don't love this method name

Reply:
This is a bespoke command for just the revision updater, it returns just the information about the revision updater. We could get the information in separate methods, but that leads to potato programming.

Note: I am open to other naming options.